crate_universe/src/metadata.rs

```diff
@@ -441,9 +442,9 @@ where
         // We expect the crate id (parts[1]) to be either
         // "<crate name> v<crate version>" or
         // "<crate name> v<crate version> (<path>)"
-        // https://github.com/rust-lang/cargo/blob/7fb01c68c190c28a43581ae7c719dcf057ecc002/src/cargo/core/package_id.rs#L210-L220
+        // "<crate name> v<crate version> (proc-macro) (<path>)"
```

Comment:
is https://github.com/rust-lang/cargo/blob/7fb01c68c190c28a43581ae7c719dcf057ecc002/src/cargo/core/package_id.rs#L210-L220 still not useful information?

Reply:
Added link to the cargo tree code